sklearn/feature_selection/rfe.py

```diff
@@ -431,14 +440,15 @@ class RFECV(RFE, MetaEstimatorMixin):
            for cancer classification using support vector machines",
            Mach. Learn., 46(1-3), 389--422, 2002.
     """
-    def __init__(self, estimator, step=1, cv='warn', scoring=None, verbose=0,
-                 n_jobs=1):
+    def __init__(self, estimator, step=1, min_features_to_select=1, cv='warn',
+                 scoring=None, verbose=0, n_jobs=1):
```

Comment:
Technically it's an API change, because for anyone who uses positional arguments `RFECV(estimator, 1, 'warn')` will have their code broken. But then I'm not sure what's out currently policy about this? Do we put the added parameter at the end (after `verbose` and `n_jobs`) or insert  earlier as done here @jnothman ?

Reply:
I had this same concern; @jnothman addressed it above. As per the docs, backwards compatibility isn't guaranteed for parameter ordering.